### ISSUE TITLE
1.16.3: ChatControlPro doesn't even load now..

### ISSUE THREAD

**Issue by cloakfox** [kangarko/ChatControl-Pro#2061]
https://pastebin.com/s0J2SD8C

Sigh...

No idea how to fix this, using default everything.

This is what that weird yml file looks like currently: https://pastebin.com/FGTrdTzW

**Comment by NerdyBirder** [comment]
What I have to do to get the latest version to work is to insert this line "Type: MESSAGE" into all three files in the variables section, then remove the "s and the { } s from the first line in the configs. I just followed the directions the error kept giving and eventually it worked.

**Comment by cloakfox** [comment]
Well maybe im dumb af but these instructions are very unclear for me for some reason.

It would be so great of it just works out of the box, and doesn't break every 2 updates.

**Comment by NerdyBirder** [comment]
I agree. He says that after the 9.1.0 update things will be more stable so i am hoping for it then.


Ok here are more step by step instructions for my fix:
1. Find the folder named variables
2. Open each file in there and insert this line somewhere in each file: Type: MESSAGE
3. After each of the three files has the new line go back to each file and find the line that says "Key:".  Remove the quotation marks and the curly brackets { } from the word in the line just leaving the plain word on that line.

Does this help?

**Comment by cloakfox** [comment]
It doesn't help, ill wait for this to actually get fixed. It's ridiculous that this garbage premium plugin shits itself after updates that claim customers are happy and that things are more stable now.


I can't wait to move away from this plugin.

**Comment by cloakfox** [comment]
> It doesn't help, ill wait for this to actually get fixed. It's ridiculous that this garbage premium plugin shits itself after updates that claim customers are happy and that things are more stable now.
> 
> I can't wait to move away from this plugin.

Just to confirm, I've regenerated the variables folder so all 3 files are fresh. And..... same issue.

Imagine claiming things are stable now but not test to see if they actually are.

**Comment by kangarko** [comment]
I understand you were frustrated with the error message and the fact you faced repetitive issues with our software to which I apology, but I am not going to put up with your shit here. 

That being said, I made the "Please set 'Type' as variable type (available: FORMAT, MESSAGE)" and other errors from variables/ not throw an exception and made other related error messages more clear.

**Comment by cloakfox** [comment]
> I understand you were frustrated with the error message and the fact you faced repetitive issues with our software to which I apology, but I am not going to put up with your shit here.
> 
> That being said, I made the "Please set 'Type' as variable type (available: FORMAT, MESSAGE)" and other errors from variables/ not throw an exception and made other related error messages more clear.

When are you planning to release an actual working version of ChatControlPro to your customers? Currently back on 8.8.16 that works but spams console like crazy with errors that got fixed in later versions but can't test since latest versions are broken.

"but we'll not tolerate using review as ransom and insulting our customer service."

Also I don't know how you came to the conclusion that I've insulted your customer support or how i'm using a review to hold you ransom to get something fixed? I finally decided to make a review because I feel like others should know that the current version doesn't even load.

It's disappointing that you have put up with **MY shit**, when all I did was report a game-breaking issue with your product. Imagine what your customers have to put up with lol.
